src/radical/entk/execman/rp/task_manager.py

```diff
@@ -313,7 +314,9 @@ def unit_state_cb(unit, state):
 
                     self._advance(task, 'Task', states.SUBMITTING,
                                   mq_channel, '%s-tmgr-to-sync' % self._sid)
-
+                    self._log.debug('Connection State %s channel state %s'
+                                        % (mq_connection.is_open,
+                                           mq_channel.is_open))
```

Comment:
I would suggest to disable that message - it's on a hot code path.  OTOH, it's `DEBUG` level, and we should not encourage to run `DEBUG` as default level either.  Hmm, so probably ok... :-)